packages/components/Link/src/Link.tsx

```diff
@@ -10,11 +10,11 @@ import { foregroundColorTokens, textTokens, borderTokens } from '@fluentui-react
 import { useAsPressable, useKeyCallback, useOnPressWithFocus, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
 import { mergeSettings } from '@uifabricshared/foundation-settings';
 import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
-import { IViewProps } from '@fluentui-react-native/adapters';
+import { ITextProps } from '@fluentui-react-native/text';
 
-export type ILinkHooks = [IWithLinkOptions<IViewProps>, ILinkState];
+export type ILinkHooks = [IWithLinkOptions<ITextProps>, ILinkState];
 
-export function useAsLink(userProps: IWithLinkOptions<IViewProps>, ref: React.RefObject<any>): ILinkHooks {
+export function useAsLink(userProps: IWithLinkOptions<ITextProps>, ref: React.RefObject<any>): ILinkHooks {
   const { url, onPress, ...rest } = userProps;
```

Comment:
Why is this changed to `ITextProps`?

Reply:
Oh, I see that `ILinkProps` is defined the same way (`IWithLinkOptions<ITextProps>`), so I'm guessing you're matching that? I'm a bit confused because the props generated from this function are being passed to the root View (see lines 59 and 71). So I'm wondering how View makes use of `ITextProps`.

Reply:
There was a mismatch in properties with the new changes to IViewProps that broke the bundle, so I tried something new. 

My logic was: When we call `useAsLink(rest, componentRef)` on line 59, the parameter `rest` is of type `ILinkProps extends IWithLinkOptions<ITextProps>`. So, even before this change, the only props that could have been passed to the root were within `IWithLinkOptions<ITextProps>`. There doesn't seem to be anything special in useAsLink that uses ViewProps. I was actually a bit confused why there were ViewProps there in the first place. Maybe I'm missing something?

Reply:
I think maybe useAsLink was using ViewProps for the `onKeyUp` property. And the reason why it works with ITextProps is because we added `onKeyUp` to ITextWin32Props?

Reply:
I think you're right. To be safe though I changes the adapters package a little so Link doesn't have to change. There was one prop that was Windows-specific that was causing an issue that I simply omitted from adding. Everything should still work correctly,